packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -2298,7 +2250,8 @@ function getValuesFromSizeObjects(obj: Record<string, Size>) {
 }
 
 function makeCssProperty(name: string) {
-  return `--${namespaced(`ScrollArea${ucFirst(name)}`)}`;
+  const cssProp = name.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
+  return `--radix-scroll-area-${cssProp}`;
 }
```

Comment:
I don't love all this abstraction, like I was saying the other day it makes it hard to see where the custom properties are etc. For now I've inlined what I could, didn't want to spend to much time in here.